.travis.yml

```diff
@@ -37,3 +23,54 @@ jobs:
         - make test-coverage
       after_success:
         - codecov
+
+    - stage: setup-cloudigrade
+      install:
+        - ./scripts/setup_oc.sh
+        - export PATH="${HOME}/oc:${PATH}"
+        - oc version
+      script:
+        - oc login $OPENSHIFT_URL --token=$OPENSHIFT_TOKEN
+        - REVISION=$(oc rollout latest dc/postgresql -o revision)
```

Comment:
To solve the issue with the job crashing and burning if its run while the old rollout is still happening, we can insert a `oc rollout status dc/postgresql` before this line, and `oc rollout status dc/cloudigrade` before line 36.